src/main/kotlin/no/nav/syfo/job/VarselSender.kt

```diff
@@ -20,42 +20,43 @@ class VarselSender(
     private val databaseAccess: DatabaseInterface,
     private val sendVarselService: SendVarselService,
     private val toggles: ToggleEnv,
-    private val appEnv: AppEnv
+    private val appEnv: AppEnv,
 ) {
-    private val log = LoggerFactory.getLogger("no.nav.syfo.job.SendVarslerJobb")
+    private val LOG = LoggerFactory.getLogger("no.nav.syfo.job.SendVarslerJobb")
```

Comment:
Er denne navneendringen pga noen warning e.l. fra intellij eller linting?
Jeg ser at vi har litt forskjellig bruk av `LOG` og `log`, men definitivt mest av `log` i koden fra før av, så det føles ikke helt riktig å rename bare i denne filen. Er `log` virkelig en konstant?

Her er forøvrig lenke til navnestandard: https://kotlinlang.org/docs/coding-conventions.html#property-names

Reply:
Ja, det var tanken bak deenne endringen, ønsket å bidra til mer konsistent bruk av `LOG` vs `log` da det er begge som er i bruk per i dag. 
Hvis du mener `const val`, nei, den er det ikke da det er kun primitiver som kan ha const modifier, altså ikke `Logger`. 
Skal revertere denne og endre de andre stedene hvor vi har `LOG`